code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -230,5 +230,24 @@
 		//to_chat(world, "post [temperature], [cached_gases["plasma"][MOLES]], [cached_gases["co2"][MOLES]])
 		return REACTING
 
+//applies the frozen visual to things under FREEZE_TEMPERATURE
+/datum/gas_reaction/freezing
+	name = "Freezing"
+	id = "freezing"
+	priority = 0
+
+/datum/gas_reaction/freezing/init_reqs()
+	max_requirements = list(
+		"TEMP" = FREEZE_TEMPERATURE
```

Comment:
Freon doesn't even change air temperature, does it? Changing this means that freon does basically nothing. Releasing it won't actually freeze things any more. I don't agree with that; even though I don't like freon you're removing its primary function. If you're gonna do this just remove the gas all the way